dev.Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM node:16 as build
+
+# Create app directory
+WORKDIR /src
+# Required files are whitelisted in dockerignore
+COPY . ./
+RUN npm ci && npm run build 
+
+EXPOSE 3000
```

Comment:
Since the port mappings are done by docker-compose in this case and the `EXPOSE` keyword is purely informative anyway, I would delete this line.

Reply:
I would keep it, as it can be also used with docker - docker compose does overwrite if necessary